src/components/List.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-const List = ({ token, results }) => {
+const List = ({ results, setSearchTerm, searchTerm }) => {
   return (
     <>
+      <form>
+        <label htmlFor="searchField">Search</label>
+        <br />
+        <input
+          onChange={event => setSearchTerm(event.target.value)}
+          value={searchTerm}
+          id="searchField"
+        ></input>
+        <button disabled={searchTerm === ''} onClick={() => setSearchTerm('')}>
```

Comment:
Should `event.preventDefault` be added to this button to prevent the component from rerendering?

Reply:
I'm not sure we need it. I know that if we submit the form we'd wanna prevent the browser's default behavior, but since we don't submit I don't think we need to include it. 

I am a bit unsure about the usage of a form here though. Maybe it isn't the right element since we never actually submit the form? I had thought that it would make the HTML more accessible but I'm not totally sure on the protocol here.

Reply:
I don't think you need a form. Here are the wire frames: https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-populated.png

Reply:
This looks great and works perfectly!  One small thing - can we add a placeholder value?  Something like "Search..."?  Then we could get rid of the "Search" Label (change it to sr-only), and I think it would look a little cleaner.  Later on when we work on styling, I'd love to find a way to have the list background not move on search.

Reply:
I like this idea! What do you think @TianaJohnson? I think it may be useful to have some type of header there, maybe not "Search", but we can always figure that out later on!

Oh! I've been waiting for us to get to the more CSS heavy Github issues, but we should definitely give it a height to prevent the current jumpiness.